rd_ui/app/views/query.html

```diff
@@ -91,6 +91,9 @@ <h4 class="modal-title">Query Archive</h4>
     <div class="col-lg-12 p-5 bg-orange c-white" ng-if="query.is_archived">
       This query is archived and can't be used in dashboards, and won't appear in search results.
     </div>
+    <div class="col-lg-12 p-5 bg-orange c-white" ng-if="query.is_draft">
+      This query is a draft.
```

Comment:
I think it will be nicer to have an icon next to the query or dashboard name rather than the message. Maybe a pencil (`fa fa-pencil`)?